README.md

```diff
@@ -90,11 +90,13 @@ myChart
         data: [
           {
             timeRange: [<date>, <date>],
-            val: <val: number (continuous dataScale) or string (ordinal dataScale)> 
+            val: <val: number (continuous dataScale) or string (ordinal dataScale)>,
+            data: {<any additional arbitrary data>}
```

Comment:
Since we're grabbing all the attributes in the object, there's no need to add this line.